test/Functional/ApplicatorTest.php

```diff
@@ -22,17 +22,6 @@ public function test_it_should_apply_value_as_a_argument_to_a_function(
         );
     }
 
-    /**
-     * @expectedException \ArgumentCountError
```

Comment:
This test was removed since the `\ArgumentCountError` will only be thrown >= 7.1